### DISCUSSION TITLE
When createing a new forge database, assignee is limitied to 10000 rows

### DISCUSSION THREAD

**Discussion by Ciunas-Bennett** [magit/forge#540] [Support] [answered]
magit-version 0ef98ef

Hi I am trying to add a new repos to the forge database. And I cannot see any PRs that are assigned to me. Looking at the database it appears that the assignees table is limited to 10000, (I am not in the list in the database) there is over 45000 people  listed as assignees in the repos I am using. Is it possible to extend the amount of rows that forge saves in the  database?

**Comment by tarsius** [comment, marked as answer]
> 45000 people listed as assignees in the repos

What repository is that?! (You might run into other issues with that repository, `forge` wasn't designed to deal with the monorepo of a large corporation.)

> Looking at the database it appears that the assignees table is limited to 10000

I don't think `forge`, `closql` or `emacsql` impose such a limit. A quick web search also did not reveal anything about SQLite having such a tiny limit, though there is a theoretical limit of 18446744073709551616 rows. :grin: https://www.sqlite.org/limits.html

I'm not ruling out that there is an issue with one of my packages, but it seems more likely that github/gitlab is imposing some limit.

**Reply by Ciunas-Bennett** [reply]
Thanks for taking your time to respond :)
I understand forge was not designed for such large repos, but I like using it so its worth giving it a try.
If the limitation is GitHub related, I was thinking of trying to insert my own details into the assignee table.
I think it is possible to figure out my ID (and forge_id) from some PR that is  assigned to me.

There also does seem to be  information missing compared to when I use forge on my own personal projects. 
For e.g. labels do not seem show up at all.
But I think I can deal with what is missing, it still is useful to me.
